panel/src/components/Sections/Sections.vue

```diff
@@ -17,7 +17,6 @@
             :is="'k-' + section.type + '-section'"
             v-if="exists(section.type)"
             :key="parent + '-column-' + columnIndex + '-section-' + sectionIndex + '-' + blueprint"
-            :blueprint="blueprint"
```

Comment:
Is there a reason why the blueprint is no longer passed down?

Reply:
As far as I saw, none of the sections actually uses it

Reply:
That's true. We define the blueprint prop in the section mixin though, in case a section wants to use it.

Reply:
True, on the other hand the value of `blueprint` is changed with this PR as well (tbh I didn't quite get whether the old behavior - name field value from the blueprint - was a bug or intended, but it seemed odd for what we were trying to use it for).

Reply:
You are right. I don't know either what it was used for at all 🤣